packages/dd-trace/src/appsec/waf/waf_context_wrapper.js

```diff
@@ -47,7 +47,7 @@ class WAFContextWrapper {
       })
 
       if (ruleTriggered) {
-        Reporter.reportAttack(result.data)
+        Reporter.reportAttack(JSON.stringify(result.events))
```

Comment:
We could do a cleaner job in serializing attacks by storing their objects then only doing a json.stringify() at the end of the request. For now we do a weird JSON manual concatenation in `reporter.js` line 132.
It would be quite some work to do this change so if you prefere we could keep it a quick change like that for now, I don't mind either way

Reply:
I was tempted to change it, but I decided against it. I can take a look at it

Reply:
No ok I guess it's fine for now, it won't be slower than it already is, we can always optimise it later